app/service/rest.py

```diff
@@ -226,8 +226,15 @@ def create_service():
     # unpack valid json into service object
     valid_service = Service.from_json(data)
 
+    # Grabbing flag from request object for default sms sender
```

Comment:
Let's add just a bit more clarification here to make sure it's clear what's happening:

```suggestion
    # Grabbing flag from request object for default SMS sender.
    # This will only be true in our tests now; the value will default to false in normal app usage.
```